test/functional/mempool_accept.py

```diff
@@ -139,6 +122,7 @@ def run_test(self):
         self.log.info('A transaction that conflicts with an unconfirmed tx')
         # Send the transaction that replaces the mempool transaction and opts out of replaceability
         node.sendrawtransaction(hexstring=tx.serialize().hex(), maxfeerate=0)
+        self.mempool_size += 1
```

Comment:
Why do we +1 here?

Reply:
I believe it's because we're keeping track of the `mempool_size` throughout the test.  
This is where we initialize it. 
`self.log.info('Start with empty mempool, and 200 blocks')
 self.mempool_size = 0` 
Also I didn't wanted to disturb the previous behavior of the test file, therefore I just kept it there :)

Reply:
oops that was my bad 😅. However, I just pushed a new commit and fixed the issue. Not the most eloquent solution as, but I'd highly appreciate any suggestions to improve it